lua-resty-openidc-1.5.4-1.rockspec

```diff
@@ -25,7 +25,7 @@ dependencies = {
     "lua >= 5.1",
     "lua-resty-http >= 0.08",
     "lua-resty-session >= 2.8",
-    "lua-resty-jwt >= 0.1.5",
+    "lua-resty-jwt >= 0.2.0",
     "lua-resty-hmac"
```

Comment:
I guess because lua-resty-jwt comes with its own (/vendor) version of lua-resty-hmac, this dependency can be removed (Travis CI would verify this)

Reply:
My thoughts exactly.